examples/quickstart-python/quickstart.py

```diff
@@ -17,117 +17,120 @@
 
-print(f'Registered device {deviceId}')
+print(f'Registered device {device_id}')
 
 # Set Device Password
-devicePassword = "my-secret-password"
-
-code = requests.put(f'http://{registryIp}:28080/v1/credentials/{tenantId}/{deviceId}',
-                    headers={'content-type': 'application/json'},
-                    data=json.dumps(
-                        [{"type": "hashed-password", "auth-id": deviceId, "secrets": [{"pwd-plain": devicePassword}]}]))
+code = requests.put(
+    url=f"{registry_base_url}/credentials/{tenant_id}/{device_id}",
+    headers={"content-type": "application/json"},
+    data=json.dumps(
+        [{"type": "hashed-password", "auth-id": device_id, "secrets": [{"pwd-plain": DEVICE_PASSWORD}]}]))
 
 if code.status_code == 204:
     print("Password is set!")
 else:
-    print("Unnable to set Password")
+    print("Unable to set Password")
 
 # Now we can start the client application
 print("You could now start the Hono Command Line Client in another terminal to consume messages from devices:")
 print()
-cmd = f'java -jar hono-cli-2.*-exec.jar app --sandbox --amqp consume --tenant={tenantId}'
+cmd = f"java -jar hono-cli-2.*-exec.jar app --sandbox --ca-file {CA_FILE} consume --tenant={tenant_id}"
 print(cmd)
 print()
 
 
 # input("Press Enter to continue...")
 
-class AmqpHandler(MessagingHandler):
-    """
-    Handler for "northbound side" where Messages are received
-    via AMQP.
-    """
-    def __init__(self, server, address):
-        super(AmqpHandler, self).__init__()
-        self.server = server
-        self.address = address
-
-    def on_start(self, event):
-        # the password used below is the Sandbox server's password for the default consumer
-        # as documented on the Hono website
-        conn = event.container.connect(self.server, user="consumer@HONO", password="verysecret") # nosec hardcoded_password_funcarg
-        event.container.create_receiver(conn, self.address)
-
-    def on_connection_error(self, event):
-        print("Connection Error")
-
-    def on_link_error(self, event):
-        print("Link Error")
-
-    def on_message(self, event):
-        print("Got a message:")
-        print(event.message.body)
-
-
-# Prepare the container
-uri = f'amqp://{amqpNetworkIp}:15672'
-address = f'telemetry/{tenantId}'
-print("Using source: " + uri)
-print("Using address: " + address)
-container = Container(AmqpHandler(uri, address))
-
-# run container in separate thread
-print("Starting (north bound) AMQP Connection...")
-thread = threading.Thread(target=lambda: container.run(), daemon=True)
-thread.start()
-
-# Give it some time to link
-time.sleep(2)
+class DownstreamApp(threading.Thread):
+    def __init__(self):
+        threading.Thread.__init__(self)
+        self.stop_event = threading.Event()
+        self._topic_name = f"hono.telemetry.{tenant_id}"
+
+    def stop(self):
+        self.stop_event.set()
+
+    def run(self):
+
+        consumer = KafkaConsumer(
```

Comment:
When I tested this, I got a 503 error when sending a message
```
created Kafka consumer ...
waiting for Kafka messages
Sending Telemetry message via HTTP adapter
failed to send message via HTTP adapter, status code: 503
Sending Telemetry message via MQTT adapter
received message: {"temp": 17, "transport": "mqtt"}
```
This meant, the python script didn't finish.

Therefore, I think it would be good to add a general timeout by means of a `KafkaConsumer` `consumer_timeout_ms` parameter here.